code/modules/mob/living/carbon/examine.dm

```diff
@@ -50,9 +50,9 @@
 			else
 				msg += "[t_He] [t_has] minor bruising.\n"
 		else
-			if(prob(33))
+			if(prob(50))
```

Comment:
really i only care if each description has an equal chance of appearing so isn't setting them all to 50 fine?